bolt-design-system/bolt
# IE and Edge ordered list background and numbering fix

## Jira

http://vjira2:8080/browse/BDS-904

## Summary

The ordered list was rendered not appropriate on Internet Explorer and Edge.

## Details

Update javascript logic to be work with above browsers. I don't know if this is the correct approach but is working.

## How to test

Run this code locally. Open pattern lab website on  Internet Explorer 11 and/or Microsoft Edge and go to: `Components -> Ordered List -> View All` and `Components -> Unordered List -> View All`. Please check if ordered list i rendered correctly (the same as in Chrome).
